modules/apps/headless/headless-admin-content/headless-admin-content-impl/rest-openapi.yaml

```diff
@@ -143,6 +143,45 @@ components:
                     description:
                         Internationalized field of the robots of the page that renders the Display Page Template
                     type: object
+        Status:
+            # @review
+            description:
+                Represents the status of a resource
+            properties:
+                code:
+                    # @review
+                    description:
+                        A number which identifies a specific status
+                    format: int32
+                    readOnly: true
+                    type: integer
+                label:
+                    # @review
+                    description:
+                        Label associated to the status
+                    readOnly: true
+                    type: string
+                label_i18n:
+                    additionalProperties:
+                        type: string
+                    readOnly: true
+                    type: object
+        Version:
+            properties:
+                number:
```

Comment:
number is a bit generic, right?

Reply:
It was the name proposed by Brian 😅

Reply:
Then LGTM. But it can create issues with java code generation because Number is a reserved keyword.